cmd/controlplane/controller.go

```diff
@@ -3,12 +3,13 @@ package main
 import (
 	"context"
 	"fmt"
+	"runtime"
```

Comment:
```suggestion
	stdruntime "runtime"
```

might be better, as we utilize `k8s.io/apimachinery/pkg/runtime` more than the stdlib.

Reply:
That sgtm.